include/irr/builtin/material_compiler/glsl/rasterization/impl.glsl

```diff
@@ -187,6 +197,9 @@ void instr_eval_execute(in instr_t instr, inout irr_glsl_LightSample s, inout ir
 	} else
 #endif
 	{} //else "empty braces"
+
+	if (op_isBXDForBlend(op))
+		writeReg(REG_DST(regs), cosFactor*result);
```

Comment:
what is `cosFactor` doing here?

Reply:
oh, you're right, it's already in the result. my bad